compiler/passes/scopeResolve.cpp

```diff
@@ -1490,33 +1490,29 @@ static void resolveModuleCall(CallExpr* call, Vec<UnresolvedSymExpr*>& skipSet)
           }
         }
 
```

Comment:
I was initially thrown off by the code starting at line 1480 (which you did not write).  (a) the comment at line 1484 doesn't have anything to do with the code that comes next (b) it would be easy for the incautious reader to assume
that entry and mbr_name are used beyond 1492.

1) I know I would be tempted to turn those few statements into a function that returns a Symbol*.
2) But baring that could you (i) fix the comment (ii) move entry and mbr_name in to the scope of the if?

I think that will clarify that the first step is to determine if the identifier can be mapped to a top-level module symbol.